presto-testing/src/main/java/io/prestosql/testing/TestBaseConnectorTest.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.testing;
+
+import com.google.common.collect.ImmutableList;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static com.google.common.collect.ImmutableList.toImmutableList;
+import static java.util.Objects.requireNonNull;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestBaseConnectorTest
+{
+    @Test
+    public void testThatAllTestsAreCopied()
+    {
+        assertThat(getMethods(BaseConnectorTest.class)).containsAll(getMethods(AbstractTestIntegrationSmokeTest.class));
+    }
+
+    private List<MethodSignature> getMethods(Class<?> clazz)
+    {
+        return Arrays.stream(clazz.getMethods())
+                .map(method -> new MethodSignature(method.getName(), method.getParameterTypes()))
+                .collect(toImmutableList());
+    }
+
+    private static final class MethodSignature
```

Comment:
This could be just a `String`. It's fine to leave as-is, as it's for transition period only.